private/tools/java/com/github/bazelbuild/rules_jvm_external/maven/MavenSigning.java

```diff
@@ -234,11 +234,15 @@ protected enum SigningMethod {
     NONE
   }
 
-  protected static class SigningMetadata {
-    protected final String signingKey;
-    protected final String signingPassword;
+  public static class SigningMetadata {
+    public final String signingKey;
```

Comment:
Use IJ to generate getters, please. We shouldn't really be messing with fields directly.